docs/integration/solrcloud.md

```diff
@@ -1,9 +1,112 @@
 title: SolrCloud Monitoring Integration
 description:  Our monitoring and logging platform includes integration for SolrCloud. Use predefined key metrics reports combined with rich data visualization tools to monitor critical issues on your Solr machines cluster, and receive alerts on memory usage, uptime, load averages, index stats, document and filter caches, latency, rate, and more
 
+Sematext offers simple and versatile SolrCloud monitoring agent written in Java and Golang without CPU and memory overhead on your application. It's easy to install and require no changes in SolrCloud source code or your application source code. 
+
+## Sematext SolrCloud Monitoring Agent 
+This lightweight, open-source [Java Monitoring Agent](https://github.com/sematext/sematext-agent-java) collects SolrCloud performance metrics and sends them to Sematext. The second part of the Sematext SolrCloud Monitoring Agent is a Golang based agent responsible for Operating System level metrics like network, I/O and more. 
+
+The Sematext SolrCloud Monitoring Agent can be run in two different modes - *in-process* and *standalone*. The *in-process* one is run as a Java agent, it is simpler to initially setup, but will require restring your Solr node when you will want to upgrade your monitoring Agent, i.e. to get new features. The benefit of *standalone* agent mode is that it is running as a separate process and doesn't require Solr restart when it is upgraded.
```

Comment:
The in-process one is run as a Java agent.* It is simpler to initially **set up** *, but will require **restarting** * your Solr node when you want to upgrade your monitoring Agent, i.e. to get new features. The benefit of **the** * standalone agent mode is that **it's** * running as a separate process and doesn't require **a** * Solr restart when **it's** * upgraded.

Reply:
Adjusted.